Code/GraphMol/Depictor/DepictUtils.cpp

```diff
@@ -263,26 +249,21 @@ RDKit::INT_VECT findNextRingToEmbed(const RDKit::INT_VECT &doneRings,
     }
     currRingId++;
```

Comment:
```suggestion
    ++currRingId;
```

Reply:
FWIW: I'm adding these suggestions because they don't hurt, but the post- versus pre-increment thing doesn't really make a difference with integers or other types which are basically free to copy.